openarena-q3e

```diff
@@ -1,29 +1,56 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
+
+# Set the necessary variables
+OA_DIR="${XDG_DATA_HOME}"
+OA_CVARS="+set fs_basegame baseoa +set protocol 71 +set fs_homepath ${OA_DIR}"
 DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
```

Comment:
I already pointed out that this should be defined as `$XDG_CONFIG_HOME/cfg`. It is not guaranteed that the current .var/app/etc. path will work in the future while the standard `$XDG_CONFIG_HOME` is future-proof.